### PR TITLE
Update code settings.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -16,5 +18,23 @@
...
+    "beautify.language": {
+        "html": [
+            "jinja-html"
+        ]
+    },
+    "[jinja-html]": {
+        "editor.formatOnSave": true,
+        "editor.defaultFormatter": "HookyQR.beautify"
+    }
+}
```
Comment (LeoMcA) [inline .vscode/settings.json:40]: Is there a vscode setting we can add to get it to add newlines to the ends of files (like this one).

File: .vscode/settings.json [hunk 2]
```diff
@@ -16,5 +18,23 @@
     ],
     "python.analysis.extraPaths": [
         "/vendor"
-    ]
-}
+    ],
+    "files.associations": {
+        "*.html": "jinja-html"
```
Comment (LeoMcA) [inline .vscode/settings.json:23]: Do we need to add this extension to the devcontainer config?

File: .vscode/settings.json [hunk 3]
```diff
@@ -16,5 +18,23 @@
...
+        "jinja-html"
+    ],
+    "beautify.language": {
+        "html": [
+            "jinja-html"
+        ]
+    },
+    "[jinja-html]": {
+        "editor.formatOnSave": true,
+        "editor.defaultFormatter": "HookyQR.beautify"
```
Comment (LeoMcA) [inline .vscode/settings.json:38]: Do we need to add this extension to the devcontainer config as well?

**Review by LeoMcA — APPROVED** [review]
Nice! r+wc

File: .vscode/settings.json (hunk 1 above)
Comment (akatsoulas) [reply .vscode/settings.json:43]: Added it in the settings

File: .vscode/settings.json (hunk 3 above)
Comment (akatsoulas) [reply .vscode/settings.json:38]: Added in devcontainer

File: .vscode/settings.json (hunk 2 above)
Comment (akatsoulas) [reply .vscode/settings.json:23]: Added in devcontainer

**Pull request merged**